x-pack/plugins/security_solution/public/timelines/components/row_renderers_browser/examples/system.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { createGenericSystemRowRenderer } from '../../timeline/body/renderers/system/generic_row_renderer';
+import { mockEndgameTerminationEvent } from '../../../../common/mock/mock_endgame_ecs_data';
+import { ROW_RENDERER_BROWSER_EXAMPLE_TIMELINE_ID } from '../constants';
+
+const SystemExampleComponent: React.FC = () => {
+  const systemRowRenderer = createGenericSystemRowRenderer({
+    actionName: 'termination_event',
+    text: 'terminated process',
```

Comment:
consider using the `i18n` `TERMINATED_PROCESS` exported from `x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/system/translations.ts`